tensorflow/python/lib/io/tf_record.py

```diff
@@ -69,7 +69,7 @@ def __init__(self,
     Leaving an option as `None` allows C++ to set a reasonable default.
 
     Args:
-      compression_type: `"GZIP"`, "ZLIB" or `"NONE"`.
+      compression_type: `"GZIP"`, "ZLIB", or `""` (no compression).
```

Comment:
Change `"ZLIB"` to `` `"ZLIB"` ``, while you're here?

Reply:
@wchargin The docstring has been updated. Thanks for the help!